demos/python_demos/rag_chatbot/servable_stream/graph.pbtxt

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+input_stream: "OVMS_PY_TENSOR:pre_prompt"
+output_stream: "OVMS_PY_TENSOR_1:completion"
+output_stream: "OVMS_PY_TENSOR_2:end_signal"
+
+node {
+  name: "pythonNode"
```

Comment:
Maybe call it `ragNode`? (would make more sense if visualized I suppose)

Reply:
I think we should follow the convention from previous demos, all demos have exactly the same config.json and similar pbtxt